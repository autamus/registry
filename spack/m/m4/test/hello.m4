define(NAME, World)
dnl This line should not show up
// macro is ifdef(`NAME', , not)defined
Hello, NAME!
