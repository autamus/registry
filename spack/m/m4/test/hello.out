
// macro is defined
Hello, World!
